real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/Gui.java

```diff
@@ -55,6 +55,7 @@ public static Gui getInstance() {
 
 	//The windowList is a list of all of the windows that are a part of the gui
 	private ArrayList<RobobuggyJFrame> windowList = new ArrayList<RobobuggyJFrame>();
+	private MainGuiWindow mainGuiWindow;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2041124219/issues/source?bid=2938665&fileBranchId=3128705#l58)